remoteworkitem/github.go

```diff
@@ -35,7 +36,8 @@ func (f *githubIssueFetcher) listIssues(query string, opts *github.SearchOptions
 func (g *GithubTracker) Fetch() chan TrackerItemContent {
 	configuration, err := config.GetConfigurationData()
 	if err != nil {
-		panic(fmt.Errorf("Failed to setup the configuration: %s", err.Error()))
+		fmt.Print(errors.Wrap(err, "Failed to setup the configuration"))
```

Comment:
Hi @kwk ,
I would like to take the error handling here in a different PR/issue - had a discussion with @baijum https://github.com/almighty/almighty-core/issues/720